unitTests/ui/ComponentEventsTest.ts

```diff
@@ -27,6 +28,43 @@ export function ComponentEventsTest() {
       $$(test.env.root).trigger('foo', { bar: 'baz' });
       expect(spy).toHaveBeenCalledWith({ bar: 'baz' });
     });
+    describe('when the event is  wrapped by jQuery and does not have any param', () => {
+      beforeAll(() => {
+        window['Coveo']['$'] = $;
+      });
+
+      beforeEach(() => {
+        test.cmp.enable();
+        test.cmp.bind.on(test.env.root, 'click', spy);
+      });
+
+      it('when triggering a native click, it calls the spy with native event as the only parameter', function() {
+        test.env.root.click();
+
+        expect(spy).toHaveBeenCalled();
+        expect(spy.calls.argsFor(0)[0] instanceof MouseEvent).toBeTruthy();
+        expect((spy.calls.argsFor(0)[0] as MouseEvent).type).toBe('click');
+      });
+
+      it('when triggering a JQuery click event without params, it calls the spy without any params', () => {
+        $(test.env.root).click();
+
+        expect(spy).toHaveBeenCalled();
+        expect(spy.calls.argsFor(1).length).toBe(0);
```

Comment:
Is the `1` passed to `argsFor` the call number? If so, I prefer to re-instantiate the spy in the `beforeEach` and use `0` rather than having the tests be order-dependent.

Reply:
It should have been 0 indeed 🤦‍♂